app/components/data/details/data-container-files.component.ts

```diff
@@ -66,44 +66,19 @@ export class DataContainerFilesComponent implements OnDestroy {
 
     @autobind()
     public handleFileDelete(files: File[]) {
-        const fileCount = files.length;
-        const taskTitle = `Delete ${fileCount} files from ${this.container.name}`;
-
-        return this.backgroundTaskService.startTask(taskTitle, (task) => {
-            let deleted = 0;
-            const blobCache = this.storageService.getBlobFileCache({
-                container: this.container.id,
-            });
-
-            // NOTE: slight pause in-between deletes to ease load on storage account
-            // may or may not be a great idea.
-            const observable = Observable.interval(100).take(fileCount);
-            observable.subscribe({
-                next: (i) => {
-                    deleted++;
-                    return this.storageService.deleteBlobIfExists(this.container.id, files[i].name).subscribe({
-                        next: (response) => {
-                            task.name.next(`${taskTitle} (${deleted}/${fileCount})`);
-                            task.progress.next(deleted / fileCount * 100);
-
-                            if (response && blobCache) {
-                                blobCache.deleteItem(files[i]);
-                            }
-                        },
-                        error: (error) => {
-                            log.error("Failed to delete blob", error);
-                        },
-                    });
-                },
-                complete: () => {
-                    task.progress.next(100);
-                    // tslint:disable-next-line:max-line-length
-                    const message = `${deleted} files were successfully removed from the file group: ${this.container.name}`;
-                    this.notificationService.success("Removed files from group", message, { persist: true });
-                },
-            });
-
-            return observable;
+        // TODO: would like to get the deleted count from this call.
```

Comment:
use next instead of complete and make the service return the number of files

Reply:
do you mean the number of files(because you can just do files.length above) or the number of files that actually existsed and were deleted?